src/jepsen/mongodb/cluster.clj

```diff
@@ -0,0 +1,189 @@
+(ns jepsen.mongodb.cluster
+  "Functions for starting and stopping MongoDB deployments."
+  (:require [clojure.java.io :as io]
+            [clojure.string :as str]
+            [clojure.tools.logging :refer [info]]
+            [jepsen.control :as c]
+            [jepsen.mongodb.control :as mcontrol]
+            [jepsen.mongodb.mongo :as m]
+            [jepsen.mongodb.time :as mt]
+            [jepsen.mongodb.util :as mu]))
+
+;; Copied from the jepsen.control.util/start-daemon! function of
+;; Jepsen version 0.1.8 with the `exec` function calls replaced with
+;; `mcontrol/exec` function calls to account for the possibility of running
+;; without any virtualization.
+(defn- start-daemon!
```

Comment:
This is totally fine, but if you want to avoid the copy & paste logic, you can make `exec` an argument, and pass in mcontrol/exec or control/exec to get local and remote variants of the function.

Reply:
Yeah, that makes sense to me and would be a pretty clean way of handling the two different implementations from `jepsen.control/exec` and `jepsen.control.local/exec`. I think I'd rather (if you'll let me) not have the changes in this pull request depend on any changes in the jepsen-io/jepsen repository since we've agreed to put the `jepsen.control.local` namespace here for now.

Reply:
Yeah, of course! Just thinking down the road. :)